dir.props

```diff
@@ -21,8 +21,7 @@
   <PropertyGroup>
     <!-- Temporarily omit code analysis -->
     <SkipImportCodeAnalysisTargets>true</SkipImportCodeAnalysisTargets>
-    <!-- Hacky BuildConfiguration -->
-    <BuildConfiguration Condition="'$(BuildConfiguration)' == ''">netcoreapp1.1-$(OSEnvironment)</BuildConfiguration>
+    <BuildConfiguration Condition="'$(BuildConfiguration)' == ''">netcoreapp-$(OSEnvironment)-Debug-x64</BuildConfiguration>
```

Comment:
I don't think osenvironment will work because it hides osx .  I'm also not certain if OS will even work